OpenUtau/Strings/Strings.fr-FR.axaml

```diff
@@ -101,33 +101,33 @@
 
-  <!--<system:String x:Key="notedefaults.lyric">Lyric</system:String>-->
-  <!--<system:String x:Key="notedefaults.lyric.defaultlyric">Default Lyric</system:String>-->
+  <!--<system:String x:Key="notedefaults.lyric">Parole</system:String>-->
+  <!--<system:String x:Key="notedefaults.lyric.defaultlyric">Parole par défaaut</system:String>-->
   <!--<system:String x:Key="notedefaults.portamento">Portamento</system:String>-->
-  <!--<system:String x:Key="notedefaults.portamento.length">Length</system:String>-->
-  <!--<system:String x:Key="notedefaults.portamento.start">Start</system:String>-->
-  <!--<system:String x:Key="notedefaults.preset">Preset</system:String>-->
-  <!--<system:String x:Key="notedefaults.preset.namenew">New Preset Name</system:String>-->
-  <!--<system:String x:Key="notedefaults.preset.remove">Remove</system:String>-->
-  <!--<system:String x:Key="notedefaults.preset.remove.tooltip">Removes last applied preset.</system:String>-->
-  <!--<system:String x:Key="notedefaults.preset.save">Save</system:String>-->
-  <!--<system:String x:Key="notedefaults.preset.save.tooltip">Save current settings to a new preset.</system:String>-->
-  <!--<system:String x:Key="notedefaults.reset">Reset All Settings</system:String>-->
-  <!--<system:String x:Key="notedefaults.reset.tooltip">Reset every setting to default values.
+  <!--<system:String x:Key="notedefaults.portamento.length">Longueur </system:String>-->
+  <!--<system:String x:Key="notedefaults.portamento.start">Départ</system:String>-->
+  <!--<system:String x:Key="notedefaults.preset">Préréglage</system:String>-->
+  <!--<system:String x:Key="notedefaults.preset.namenew">Nouveau nom du préréglage</system:String>-->
+  <!--<system:String x:Key="notedefaults.preset.remove">Effacer</system:String>-->
+  <!--<system:String x:Key="notedefaults.preset.remove.tooltip">Effacer le dernier préréglage.</system:String>-->
+  <!--<system:String x:Key="notedefaults.preset.save">Sauvegarder</system:String>-->
+  <!--<system:String x:Key="notedefaults.preset.save.tooltip">Enregistrer les paramètres actuels en préréglage.</system:String>-->
+  <!--<system:String x:Key="notedefaults.reset">Réinitialiser tous les paramètres</system:String>-->
+  <!--<system:String x:Key="notedefaults.reset.tooltip">Réinitialiser tous les paramètres par défaut.
 Warning: this option removes custom presets.</system:String>-->
   <!--<system:String x:Key="notedefaults.vibrato">Vibrato</system:String>-->
   <!--<system:String x:Key="notedefaults.vibrato.autominlength">Minimum Length</system:String>-->
-  <!--<system:String x:Key="notedefaults.vibrato.autotoggle">Automatic Vibrato by Length</system:String>-->
-  <!--<system:String x:Key="notedefaults.vibrato.depth">Depth</system:String>-->
+  <!--<system:String x:Key="notedefaults.vibrato.autotoggle">Vibrato automatique selon la longueur</system:String>-->
+  <!--<system:String x:Key="notedefaults.vibrato.depth">Profondeur</system:String>-->
```

Comment:
Could maybe be "Amplitude" since it's how you describe waves. "Longueur" and "Période" are part of this vocabulary.